.github/ISSUE_TEMPLATE/crx-tut-request.yaml

```diff
@@ -0,0 +1,61 @@
+name: Extension tutorial request
+description: Use this form to request a new Chrome Extension tutorial
+title: "[EXT-TUTORIAL] <title>"
+labels: [content, extensions]
+assignees: amysteam
+body:
+- type: markdown
+  attributes:
+    value: "## Thank you for sharing your feedback. Please fill out this form so that we can better understand what extension tutorial you are looking for:"
+- type: textarea
+  attributes:
+    label: What would this extension do?
+    description: A concise description of how the extension would work.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: What lessons would this tutorial teach?
+    description: Top three extension concepts it will teach.
+    placeholder: |
+      1. ...
+      2. ...
+      3. ...    
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: What Chrome APIs should be featured?
+    description: Chrome APIs you'd like to see demonstrated.
+    placeholder: |
+      If you are not sure, please leave blank.
+  validations:
+    required: false
+- type: dropdown
+  id: components
+  attributes:
+    label: Extension components
+    description: Is there a specific component you would like to see?
+    multiple: true
+    options:
+      - Service worker
+      - Popup
+      - Overlay (content script)
+      - Omnibox
+      - Context Menu
+      - Options page
+      - Any
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Anything else?
+    description: |
+      Links? References? Anything that will give us more context about the tutorial you are requesting!
+
+      Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+  validations:
+    required: false
+- type: markdown
+  attributes:
+    value: "Thanks again for sharing your extension tutorial ideas with us!" 
```

Comment:
```suggestion
    value: "Thanks again for sharing your extension tutorial ideas with us." 
```

Reply:
Quick writing tip. Exclamation points aren't for emphasis. They're for, well, exclamations. 'Stop!' 'Wow!' 'Awesome!'